template/pyproject.toml

```diff
@@ -32,3 +32,8 @@
     # type_correct = ""
     # type_incorrect = ""
     # help_url = ""
+
+  [ tool.<your-bot-name>.autolabel ]
+
+    # enabled = true
```

Comment:
This also means that packages need to opt-in instead of opting out. I think @astrofrog was against it. Discuss.

Reply:
I could argue for either, equally. So it's up to you :)

Reply:
The main issue at the moment is that the bot configuration is done via ``pyproject.toml`` files, but I thought that adding even an empty ``pyproject.toml`` file resulted in opting in to isolated build so it's kind of a pain to have to do that just to configure the bot. But maybe @Cadair can correct me if I'm wrong?

Reply:
The configuration system can be overridden on either a bot deployment or repo level, so it should be opt-in in Baldrick and astropy-bot can turn it on by default.

I think the majority of `pyproject.toml` related issues with pip are behind us, I think they know people are now using it for stuff like this.